### DISCUSSION TITLE
AMD RX 6750XT wont boot past Sonoma version 14.0 using NootRX

### DISCUSSION THREAD

**Discussion by MCarfs01** [ChefKissInc/NootRX#61] [Q&A]
Hi all. I was wondering if anyone could shed any light on this for me. I successfully installed Sonoma 14.0 using NootRX on my 6750XT, it worked like a dream, but I have tested and tried to update to each os version in order from 14.1 - 14.5 without success. Using the latest opencore and all my kexts and extensions are up to date. The only way I can get past a boot loop is to disable the NootRX kext, which leads me to believe its an issue with Sonoma past the first version 14.0. 

I'm using a Ryzen 3700x cpu if that makes any difference. 

I'm new-ish to hackintosh as I used to have an intel hackintosh machine before opencore was even a thing using clover. Then I stopped using the hack for work reasons. but now wanting to get back into using my PC as a dual boot PC/Mac.

**Comment by VisualEhrmanntraut** [comment]
https://chefkissinc.github.io/nootrx#prerequisites

**Reply by MCarfs01** [reply]
I have followed the opencore guide and have made sure I have already followed the prerequisites. As I said, it works on 14.0 but no version higher.

**Reply by MCarfs01** [reply]
Ok, so it now works but only if I set NootRX to be the last kext in order loaded for it to work!
